### ISSUE TITLE
Fix problem with Firebase API key

### ISSUE THREAD

**Issue by DanielEFrampton** [PairedStaging/paired-fe#19]
Travis reported error when it attempted to deploy:

```
Error: HTTP Error: 401, Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.
Firebase deployment failed
failed to deploy
```

**Comment by rer7891** [comment]
Is this just a Travis issue? Is it possible you need an environmental variable in the Travis build?

**Comment by DanielEFrampton** [comment]
@rer7891 I don't think so, because I'm trying to do it the same way that worked before, but there might be something I'm missing. Here's what I've learned and tried so far:
- I followed the instructions in [this Travis doc](https://docs.travis-ci.com/user/deployment/firebase/#generating-your-firebase-token) to get a CI deployment key from the Firebase CLI and then use the Travis CLI to encrypt that key and add it to the `.travis.yml` file.
- When that didn't work, I tried adding additional flags to the Travis CLI command to specifically target the PairedStaging/paired-fe repo on travis-ci.com:
```
travis encrypt "1/AD7sdasdasdKJA824OvEFc1c89Xz2ilBlaBlaBla" -r PairedStaging/paired-fe --pro --add
```
- I also tried updating my locally installed Firebase CLI to the newest version, which didn't appear to have any effect.

**Comment by DanielEFrampton** [comment]
This was resolved by pull request #22. The problem was two-fold: 
- Because we are using the `travis-ci.com` version of Travis, which was necessary because this repo belongs to an organization, I needed to be logged into the CLI with that account using `travis login --com`. This gave me a more descriptive error message when attempting to deploy, which pointed out the next step:
- When generating a Firebase access token with `firebase login:ci` I chose the wrong Google account, assuming I had used the one I typically use for development when I apparently had used my personal account. I corrected this by inviting my other Google account to the project, granting it Owner privileges, and restarting the Travis build.
